instant-distance-py/examples/perf-fasttext.rs

```diff
@@ -0,0 +1,111 @@
+//! This is a simple example of how to use the instant-distance crate to build an index from a file.
```

Comment:
Please isolate the addition of this example in a separate commit.

Reply:
Done